Makefile

```diff
@@ -0,0 +1,56 @@
+SHELL := /bin/bash
+
+
+.PHONY: oauth-token hmac-token github-oauth-config cookie plugins update-config
+
+### Prow Compenents
+
+1password-local:
+	export $$(xargs <.env) && ./tools/deploy_prow.sh
```

Comment:
Just a question: 
why this make command also deploys prow?

Reply:
This was for local testing, so it would pull the configmaps and deploy all the peices without failing... it probably can removed for now if wanted.